Makefile

```diff
@@ -212,100 +202,101 @@ endif
 TESTGOMODULES = $(addprefix test-, $(GOMODULES))
 
 $(TESTGOMODULES):
-	cd $(dir $(@:test-%=%)) && go test $(GOTEST_OPTS) ./...
+	cd $(@:test-%=%) && go test $(GOTEST_OPTS) ./...
 
 .PHONY: test
 test: $(TESTGOMODULES) ## Run Go unit tests
 
 ##@ Docker
 
 # Export params required in Docker Bake
+BAKE_ENV = DOCKER_REGISTRY=$(DOCKER_REGISTRY)
+BAKE_ENV += DOCKER_TAG=$(DOCKER_TAG)
+BAKE_ENV += VERSION=$(VERSION)
+BAKE_ENV += BUILD_TIMESTAMP=$(BUILD_TIMESTAMP)
+BAKE_ENV += COMMIT_HASH=$(COMMIT_HASH)
+
 BAKE_OPTS =
-BAKE_OPTS += --set *.args.DOCKER_REGISTRY=$(DOCKER_REGISTRY)
-BAKE_OPTS += --set *.args.DOCKER_TAG=$(DOCKER_TAG)
-BAKE_OPTS += --set *.args.VERSION=$(VERSION)
-BAKE_OPTS += --set *.args.BUILD_TIMESTAMP=$(BUILD_TIMESTAMP)
-BAKE_OPTS += --set *.args.COMMIT_HASH=$(COMMIT_HASH)
 ifneq ($(strip $(VMCLARITY_TOOLS_BASE)),)
 	BAKE_OPTS += --set vmclarity-cli.args.VMCLARITY_TOOLS_BASE=$(VMCLARITY_TOOLS_BASE)
```

Comment:
This could also go as a BAKE_ENV as the bake file expects it to be an env var, WDYT?

Reply:
I am good with the current structure by keeping the `BAKE_OPTS` as we might need to extend the list of arguments passed to the `docker buildx bake` command.